Source/Extensions/Blazorise.DataGrid/Contexts/DetailRowTriggerContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Blazorise.DataGrid
+{
+    /// <summary>
+    /// Holds the DetailRowTrigger context configuration.
+    /// </summary>
+    /// <typeparam name="TItem"></typeparam>
+    public class DetailRowTriggerContext<TItem>
+    {
+        public DetailRowTriggerContext( TItem item )
+        {
+            Item = item;
+        }
+
+        /// <summary>
+        /// Gets the model.
+        /// </summary>
+        public TItem Item { get; private set; }
+
+        /// <summary>
+        /// If DetailRow is showing, and if the trigger is re-evaluated the DetailRow will be set to false. Defaults to <c>true</c>.
+        /// </summary>
+        public bool Toggleable { get; set; } = true;
```

Comment:
Should this be changeable from the outside? Can we make it private or internal?

Reply:
THe point is for the user to mutate yes, according to their needs. If they want they set it to false therefore having a different behaviour.
I kinda debated with myself if this should be an OptionsAPI instead...

Reply:
OK if mutability was wanted then it is fine like this.